lib/feature_management.rb

```diff
@@ -158,7 +158,9 @@ def self.idv_allow_hybrid_flow?
   end
 
   def self.idv_gpo_only?
+    outage_status = OutageStatus.new
     IdentityConfig.store.feature_idv_force_gpo_verification_enabled ||
-      OutageStatus.new.any_phone_vendor_outage?
+      outage_status.any_phone_vendor_outage? ||
+      outage_status.phone_finder_outage?
```

Comment:
Do we have documentation about when to set `feature_idv_force_gpo_verification_enabled`? Seems like it's a little complicated to set the right flags.

Reply:
No this is a new flag we've added. We are adding flags for both enabling/disabling vendor services (PhoneFinder, Pinpoint etc), but also for disabling/enabling "features" of our application -- gpo, hybrid flow, etc. This is on the heels of a conversation about what the on-call people would prefer to have. 
  
We might consider a subsequent ticket that updates runbooks with these property values and their descriptions though.